test/unit/traversing.js

```diff
@@ -743,6 +743,37 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
```

Comment:
Yes, I know in older browsers the code will get executed; you should skip the inertness test in those browsers; see [this test](https://github.com/jquery/jquery/blob/58c6ca9/test/unit/event.js#L1403) for an example how to do it.

By treating the contents of the template element not carefully you can lose the inertness; for example if you append `template.content` to the DOM. jQuery applies some manipulation steps in various APIs so we need tests that ensure that a simple `$('template').contents()` doesn't break the inertness by itself.